### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by Kaiserabbas**
## Styling to match  Zeplin's design
- My aim was to match Zeplin's design (this includes all elements + the same styling)
- I have tried to use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/BookCard.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ const BookCard = ({ book, index, onEdit, onComment }) => {
       </div>
       <div className="book-card-right">
         <p className="current-chapter">Current Chapter</p>
-        <p className="book-state">{book.state || 'Not Started'}</p>
+        <p className="book-state">{book.state || 'Chapter-1'}</p>
         <button type="button" onClick={handleUpdateProgress}>
           Update Progress
         </button>
```
Comment (V-Blaze) [inline src/components/BookCard.js:58]: - Your project looks like the provided wireframe, :+1: :+1: But there are few changes you need to make before going to the next milestone. 

- [ ] Looking at your `.eslintrc.json`, it seems you are using the wrong or modified code, The content of the file is different from the one in the official Microverse Linters repository and it affects the Linters check, some errors in the code are not picked up by linters. I'll suggest you go to this repository [ [MICROVERSE LINTERS](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) ] and copy the correct `.eslintrc.json` file content. Make sure to run the linters check after that and fix any error that your code has. NB: It is against Microverse rules to Modify any Linters code, the linters are there to make sure your code follows the best standard. :smile: :wave: :+1: 

## I'LL GREATLY SUGGEST YOU STOP MODIFING THE CONTENT OF ANY OF THE LINTERS FILES AS IT IS AGAINST THE MICROVERSE RULE.

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Kaiserabbas 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- UI looks great✔
-  Correct Git flow👋 
-  Readme looks professional ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookCard.js (hunk 1 above)
Comment (Kaiserabbas) [reply src/components/BookCard.js:58]: Thank you for your kind review and suggestions.
This issue is addressed and .eslintrc is restored back to the original copy as provided by Microverse.

**Review by rotimiazeez — APPROVED** [review]
Hi @Kaiserabbas 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good Job implementing the required change 👌🏽
- Good communication with the reviewer 👍🏽
- All linter checks passed 👏🏽
- Good PR and README file 💪🏽

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**